README.md

```diff
@@ -103,6 +104,20 @@ RNS3.put(file, options)
   .progress((e) => console.log(e.loaded / e.total));
 ```
 
+## Cognito
```

Comment:
While I definitely appreciate you adding detailed documentation, thoughts on simplifying this section? I think we can rely on Amazon's docs to describe Cognito better than we can here.

How about something like

>[Cognito](http://docs.aws.amazon.com/cognito/latest/developerguide/what-is-amazon-cognito.html) is a service that enables you to create unique identities for your users. If you are using Cognito, you'll need to pass in the session token you received from AWS Security Token Service as the `sessionToken` key in the `options` hash. See the [Cognito](http://docs.aws.amazon.com/cognito/latest/developerguide/what-is-amazon-cognito.html) and [temporary security credentials](http://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_temp_use-resources.html#RequestWithSTS) documentation for more information.